iOS/README.md

```diff
@@ -20,6 +20,36 @@ For testing purposes, you can add your device IDFA to test mode devices [here](h
 
 For a full guide refer to the Applovin integration documentation for iOS available [here](https://dash.applovin.com/documentation/mediation/ios/getting-started/integration).
 
+### Installation
```

Comment:
Perfect, can you just add a sentence at the beginning like "You can install the adapter via cocoapods or manually." to avoid the user follow both steps (we never know..)